tests/run.go

```diff
@@ -442,41 +450,49 @@ func Run(o *RunOption) {
 				command.New(o.BaseOpt, "run", "-i", "--name", testContainerName, "--mount",
 					fmt.Sprintf("type=bind,source=%s,target=%s,ro", fileDir, destDir),
 					defaultImage).WithStdin(gbytes.BufferWithBytes(cmd)).WithoutSuccessfulExit().Run()
+
+				if runtime.GOOS == "windows" {
+					fileDir, err = getWslPath(fileDir)
+					gomega.Expect(err).Should(gomega.BeNil())
+				}
 				expectedMount := []MountJSON{makeMount(bindType, fileDir, destDir, "ro", false)}
 				actualMount := getContainerMounts(o.BaseOpt, testContainerName)
 				verifyMountsInfo(actualMount, expectedMount)
 			})
 
 			ginkgo.It("should create nested bind mounts within a container", func() {
-				const (
-					outerDir  = "/outer"
-					nestedDir = "/outer/nested"
-				)
-
+				var err error
 				// Create the nested directory on the host
-				hostDirectory := ffs.CreateNestedDir(outerDir)
-				nestedDirectory := ffs.CreateNestedDir(nestedDir)
-				defer ffs.DeleteDirectory(hostDirectory)
+				containerOuterDir := ffs.CreateFilePathInHome("outer")
+				nestedHostDir := ffs.CreateNestedDir(filepath.Join("outer", "nested"))
+				nestedContainerDir := nestedHostDir
+				if runtime.GOOS == "windows" {
+					nestedContainerDir, err = getWslPath(nestedHostDir)
+					gomega.Expect(err).Should(gomega.BeNil())
+					containerOuterDir, err = getWslPath(containerOuterDir)
+					gomega.Expect(err).Should(gomega.BeNil())
+				}
+				defer ffs.DeleteDirectory(nestedHostDir)
```

Comment:
should we be removing `containerOuterDir` ?

Reply:
After fixing `CreateFilePathInHome` , we don't need to since it's not created on the host anymore and this path(`containerOuterDir`) is the mount point in the container.